.circleci/config.yml

```diff
@@ -1,13 +1,18 @@
 version: 2
 
 jobs:
-  python3:
+  python3-min-dependency-versions:
```

Comment:
Maybe call this, `doc-min-dependencies` as it shows in CI at each PR and the current name is a bit long. Everything is python3 now so putting it in the title no longer brings useful information..

Reply:
So, `doc-min-dependencies` and `doc`?